doc/mgmt/Docker to Host communication.md

```diff
@@ -34,6 +34,9 @@ Docker to Host Communications
 | 0.7 | 04/23/2020  | Mike Lazar         | Added authentication failure      |
 |     |             |                    | log information                   |
 |:---:|:-----------:|:------------------:|-----------------------------------|
+| 0.8 | 05/26/2022  | Gang Lv            | Added generic config updater      |
```

Comment:
The PR title is not accurate. It is not for specific application such as gNMI service. It is mainly on adding GCU capability on D-Bus host service.